api/src/main/resources/api/service_openapi.yaml

```diff
@@ -466,6 +471,19 @@ components:
           type: string
         filter:
           $ref: '#/components/schemas/Filter'
+
+    SearchTermFilter:
```

Comment:
The one risk of including this as part of the main filtering API is if the semantics of this are fuzzy, i.e. whether something constitutes a match could vary across versions of Tanagra, or versions of the indexing pipeline.

If it is indeed stable/portable, then this could actually be an interesting filter to have within a cohort or dataset selection. I'm trying to think whether or not it would be problematic. Probably OK being here, at least for now I think..